getting_started/devpreview_faq.adoc

```diff
@@ -20,16 +20,16 @@ during the preview period.
 [[devpreview-faq-product]]
 == Product
 
-HOW IS THE NEXT GEN DEVELOPER PREVIEW DIFFERENT THAN OPENSHIFT ONLINE?::
-{product-title} currently runs on the OpenShift 2.X platform. The Next Gen
-Developer Preview brings OpenShift 3, built on Docker and Kubernetes, to the public 
+HOW IS THE NEXT GEN DEVELOPER PREVIEW DIFFERENT THAN {product-title} ONLINE?::
+{product-title} currently runs on the {product-title} 2.X platform. The Next Gen
+Developer Preview brings {product-title} 3, built on Docker and Kubernetes, to the public
 cloud in a multi-tenant environment.
 link:../getting_started/online_v2_vs_v3.html[Learn more].
```

Comment:
@ahardin-rh Do we know if these in the caps are going to come out in upper case or lower case?